packages/actor-context-preprocess-set-defaults/lib/ActorContextPreprocessSetDefaults.ts

```diff
@@ -35,8 +39,21 @@ export class ActorContextPreprocessSetDefaults extends ActorContextPreprocess {
         .setDefault(KeysQuerySourceIdentify.sourceIds, new Map())
         .setDefault(KeysCore.log, this.logger)
         .setDefault(KeysInitQuery.functionArgumentsCache, this.defaultFunctionArgumentsCache)
-        .setDefault(KeysQuerySourceIdentify.hypermediaSourcesAggregatedStores, new Map());
+        .setDefault(KeysQuerySourceIdentify.hypermediaSourcesAggregatedStores, new Map())
+        .setDefault(KeysStatisticsTracker.statistics, new Map());
 
+      if (context.get(KeysStatisticsTracker.statisticsSaveLocation)) {
```

Comment:
Not sure if this really belongs in Comunica (it's not browser-safe). Could be a post-processing step outside of Comunica I guess?

Reply:
I also don't think CLI access is super useful.